test/sim/team-validator.js

```diff
@@ -476,42 +476,34 @@ describe('Team Validator', function () {
 
 	it('should reject Gmax Pokemon from before Gen 8', function () {
 		let team = [
-			{species: 'charizard-gmax', ability: 'blaze', moves: ['flamethrower'], evs: {hp: 1}},
+			{species: 'charizard', ability: 'blaze', moves: ['flamethrower'], evs: {hp: 1}, gigantamax: true},
 		];
 		let illegal = TeamValidator.get('gen8anythinggoes').validateTeam(team);
 		assert.equal(illegal, null);
 
 		team = [
-			{species: 'charizard-gmax', ability: 'blaze', moves: ['roost'], evs: {hp: 1}},
+			{species: 'charizard', ability: 'blaze', moves: ['roost'], evs: {hp: 1}, gigantamax: true},
 		];
 		illegal = TeamValidator.get('gen8anythinggoes').validateTeam(team);
 		assert(illegal);
 
 		team = [
-			{species: 'pikachu-gmax', ability: 'static', moves: ['thunderbolt'], evs: {hp: 1}},
+			{species: 'pikachu', ability: 'static', moves: ['thunderbolt'], evs: {hp: 1}, gigantamax: true},
 		];
 		illegal = TeamValidator.get('gen8anythinggoes').validateTeam(team);
 		assert.equal(illegal, null);
 	});
 
 	it('should reject exclusive G-Max moves added directly to a Pokemon\'s moveset', function () {
 		const team = [
-			{species: 'charizard-gmax', ability: 'blaze', moves: ['gmaxwildfire'], evs: {hp: 1}},
+			{species: 'charizard', ability: 'blaze', moves: ['gmaxwildfire'], evs: {hp: 1}, gigantamax: true},
 		];
 		let illegal = TeamValidator.get('gen8anythinggoes').validateTeam(team);
 		assert(illegal);
 		illegal = TeamValidator.get('gen8customgame@@@-nonexistent').validateTeam(team);
 		assert(illegal);
 	});
 
-	it('should reject Gmax Pokemon from formats with Dynamax Clause', function () {
```

Comment:
I see no reason to remove this test; I think it should be revised to `gigantamax: true` like the others.

Reply:
We should also keep some tests using the `species: '...-gmax'` format, to make sure the validator is correctly converting those to `gigantamax: true`.

Reply:
This test is no longer necessary since the clause doesn’t have any validation on it anymore